src/main/java/org/alliancegenome/curation_api/jobs/executors/BulkLoadJobExecutor.java

```diff
@@ -108,11 +110,12 @@ public void process(BulkLoadFile bulkLoadFile, Boolean cleanUp) throws Exception
 			ontologyExecutor.execLoad(bulkLoadFile);
 		} else if (bulkLoadFile.getBulkLoad().getBackendBulkLoadType() == BackendBulkLoadType.RESOURCE_DESCRIPTOR) {
 			resourceDescriptorExecutor.execLoad(bulkLoadFile);
+		} else if  (bulkLoadFile.getBulkLoad().getBackendBulkLoadType() == BackendBulkLoadType.EXPRESSION) {
```

Comment:
There is a weird extra space here, surprised checkstyle didn't catch it.

Reply:
Fixed